udata/core/discussions/caches.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from flask import url_for
+
+from udata.app import cache
+
+from .signals import (
+    on_new_discussion, on_new_discussion_comment, on_discussion_closed,
+    on_discussion_deleted
+)
+
+
+@on_new_discussion.connect
+@on_new_discussion_comment.connect
+@on_discussion_closed.connect
+@on_discussion_deleted.connect
+def invalidate_discussions_cache(discussion, **kwargs):
+    cache_key = 'discussions{url}?for={id}'.format(
```

Comment:
Given the previous cache key implementation, I think the key is missing the `/api/1/` part.
Why not use directly `url_for('api.discussions', qs={'for': discussion.subject.id})` ?